refs/types_test.go

```diff
@@ -18,3 +19,32 @@ func TestZeroSubnet(t *testing.T) {
 	refs.MakeZeroSubnet(id)
 	require.True(t, refs.IsZeroSubnet(id))
 }
+
+func TestSubnetID_MarshalText(t *testing.T) {
```

Comment:
`(*refs.SubnetID)(nil).MarshalText()` is also worth testing.

Reply:
Tested.